composer.json

```diff
@@ -175,6 +175,7 @@
         "google/cloud-osconfig": "1.2.0",
         "google/cloud-oslogin": "1.5.2",
         "google/cloud-policy-troubleshooter": "1.2.0",
+        "google/cloud-policysimulator": "0.0.0",
```

Comment:
This is expected because at the time of this PR being merged, there is no release, so the current release is `0.0.0`. Since you've correctly tagged this as a `feat`, the next release will pick up that a feature has been released, and increment this component to `0.1.0` and tag it once the release PR is merged.